includes/class-wc-tracker.php

```diff
@@ -111,9 +111,22 @@ private static function get_tracking_data() {
 		$data['inactive_plugins'] = $all_plugins['inactive_plugins'];
 
 		// Jetpack & WooCommerce Connect.
+		// Test if the site if a staging site based on the Jetpack criteria.
+		$is_jp_staging = false;
+		if ( class_exists( '\Automattic\Jetpack\Status' ) ) {
+			// Preferred way of checking with Jetpack 8.1+.
+			$jp_status = new \Automattic\Jetpack\Status();
+			if ( is_callable( '$jp_status->is_staging_site()' ) && $jp_status->is_staging_site() ) {
```

Comment:
I don't think that `is_callable( '$jp_status->is_staging_site()' )` works. I believe it will always return false. To check if a given method of an object is callable you should pass an array where the first item is the object and the second item is the name of the method:

```php
is_callable( array( $jp_status, 'is_staging_site' ) )
```

Reply:
You're right of course. Thanks, fixed both items in https://github.com/woocommerce/woocommerce/pull/25670/commits/88b8c006dbd92e5f64ebc35ee5658c0fcac4c0a1, I believe.